components/tools/modules/ROOT/pages/indexers/subquery.adoc

```diff
@@ -0,0 +1,27 @@
+# Intro
+
+SubQuery is a leading blockchain data indexer that provides developers with fast, flexible, universal, open source and decentralised APIs for web3 projects. SubQuery SDK allows developers to get rich indexed data and build intuitive and immersive decentralised applications in a faster and more efficient way. SubQuery supports 100+ ecosystems including Skale, Ethereum, Polygon, Polkadot, Algorand, NEAR, and Avalanche.
```

Comment:
Can you please either remove the last sentence of this paragraph or remove the other chains?